### ISSUE TITLE
Docker container missing yices-smt2

### ISSUE THREAD

**Issue by weaversa** [GaloisInc/saw-script#729]
```
$ docker run -it galoisinc/saw:0.5
 ┏━━━┓━━━┓━┓━┓━┓
 ┃ ━━┓ ╻ ┃ ┃ ┃ ┃
 ┣━━ ┃ ╻ ┃┓ ╻ ┏┛
 ┗━━━┛━┛━┛┗━┛━┛ version 0.5 (<non-dev-build>)

sawscript> prove yices {{ True }}
saw: Stack trace:
"prove" (<stdin>:1:1-1:6):
"yices" (<stdin>:1:7-1:12):
Unable to locate the executable "yices-smt2" needed to run the solver Yices
```

`yices-smt2` is a dependency for the path SAT checking option in crucible_llvm_verify.

Of course, it would be nice if cvc4, yices, abc, etc. were all available.

**Comment by jpziegler** [comment]
See the [Dockerfile](https://github.com/GaloisInc/cryptol/blob/master/Dockerfile) from GaloisInc/Cryptol to add solvers to the SAW Dockerfile.

**Comment by atomb** [comment]
Fixed in f3296d3f54.
